### DISCUSSION TITLE
What's the status on this project?

### DISCUSSION THREAD

**Discussion by frenzybiscuit** [Joshua-Riek/ubuntu-rockchip#1137] [Q&A]
I see the GitHub has been reopened and there is some ~60 updates in the package manager.

What exactly is no longer being updated/maintained on active devices? Is it just the kernel?

I'd go Armbian, but I really don't want to.

I am 100% fine with staying on 24.04 as long as possible, so long as there are security updates.

**Comment by dgdonovan** [comment]
See the announcement: #1104

The maintainer resigned due to burnout. Package updates occur during the nightly build, but there will be no kernel updates.

**Reply by frenzybiscuit** [reply]
> See the announcement: #1104
> 
> The maintainer resigned due to burnout. Package updates occur during the nightly build, but there will be no kernel updates.

Yes, I saw that. I also saw that:

A) This repo unfroze.

B) He made commits to the project.

So is he gone for good and I should look at moving to Armbian, or is it just a long break?

Right now as it currently stands the machines work and have all the packages I want/need. My main concern is security updates going forward.

**Reply by dgdonovan** [reply]
I wouldn't expect him to be back.

Armbian is probably your best option. The kernel repo (https://github.com/Joshua-Riek/linux-rockchip) that Joshua was modifying to support all the RK3588 boards, then feed into the Ubuntu build, was forked from Armbian.

**Reply by HSTe** [reply]
I still sponsor this project, and hope that Joshua will find the motivation and others will sponsor him for his great work
